docs_src/design/adr/device-service/0011-DeviceService-Rest-API.md

```diff
@@ -79,12 +79,12 @@ The values obtained when readings are taken, or used to make settings, are expre
 
 | Type | EdgeX types | Representation
 | --- | --- | ---
-| Boolean | `bool` | "true" or "false"
-| Integer | `uint8-uint64`, `int8-int64` | Numeric string, eg "-132"
-| Float | `float32`, `float64` | base64 encoded little-endian binary or decimal with exponent, eg "1.234e-5"
-| String | `string` | string
-| Binary | `bytes` | octet array
-| Array | `boolarray`, `uint8array-uint64array`, `int8array-int64array`, `float32array`, `float64array` | JSON Array, eg "["1", "34", "-5"]"
```

Comment:
I feel sorry to bring this topic back again.  However, during the implementation, lowercase is better than camel case.
For example, when I compare value type, I have to convert the value from Profile to lowercase and covert the enum constants in contract to compare whether they are matched.
If we define the constants to lowercase, we just need to convert the value from Profile.

Reply:
I'm happy to revert this back to lowercase. The specification and the implementation were out of sync, and we decided to change the spec to match, to avoid potential for breaking change. But as we are now working towards 2.0 we can re-open this question. Could be discussed at the Core WG next week or the DS WG on 7th Dec?

Reply:
resolved in DS WG 14th Dec that PascalCase would be used for the type names. Case-insensitive validation will be done so that existing data (eg device profiles) still works